packages/connect-explorer/src/data/methods/management/rebootToBootloader.ts

```diff
@@ -5,6 +5,19 @@ export default [
         url: `/method/rebootToBootloader`,
         name,
         submitButton: 'Reboot to bootloader',
-        fields: [],
+        fields: [
+            {
+                name: 'boot_command',
+                label: 'boot_command',
+                type: 'select',
+                optional: false,
+                value: 0,
+                placeholder: 'Select',
+                data: [
+                    { value: 0, label: 'STOP_AND_WAIT' },
+                    { value: 1, label: 'INSTALL_UPGRADE' },
+                ],
+            },
+        ],
```

Comment:
I would like to make some fields optional - if `INSTALL_UPGRADE` is selected, 2 more fields should be rendered, but I don't think we have some good support for this now.